src/ducks/balance/helpers.js

```diff
@@ -52,37 +51,51 @@ export const getTransactionsForAccount = (accountId, transactions) => {
   })
 }
 
+export const getDateRange = dates => {
+  return {
+    from: getEarliestDate(...dates),
```

Comment:
Nit ?

```
getEarliestDate.apply(null, dates)
```

Reply:
Actually this function is not used. I remove it.